src/libsyncengine/jobs/network/upload_session/uploadsessionlog.cpp

```diff
@@ -0,0 +1,100 @@
+/*
+ * Infomaniak kDrive - Desktop
+ * Copyright (C) 2023-2024 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "uploadsessionlog.h"
+#include "libparms/db/parmsdb.h"
+
+namespace KDC {
+
+UploadSessionLog::UploadSessionLog(const SyncPath &filepath, uint64_t nbParalleleThread /*= 1*/)
```

Comment:
I am just wondering if it would be better to have `loguploadsession` or `uploadlogsession`.

Reply:
As it is a derived class, I would prefer to have either LogUploadSession or UploadSessionLog, but not uploadlogsession, for easier identification of the type.

Reply:
I would then go for `LogUploadSession`.

Reply:
[8a4e0a](https://github.com/Infomaniak/desktop-kDrive/pull/139/commits/a8a4e0a4af7b5a20587cc5b2913bc04aa1cdcfa6)